portafly/portafly_api.yml

```diff
@@ -36,95 +36,240 @@ paths:
+        plan:
+          $ref: '#/components/schemas/Plan'
+        created_on:
           description: When it was created
+          format: date-time
           type: string
-        updated_at:
+        traffic_on:
+          description: ??
           format: date-time
-          description: When it was last updated
           type: string
-        state:
-          description: The current state of the account
+      example:
+        id: 1
+        name: Cool App
+        state: live
+        account:
+          $ref: '#/components/examples/accountExample'
+        plan:
+          $ref: '#/components/examples/planExample'
+        created_on: 2020-06-01T00:00:00Z
+        traffic_on: 2020-06-01T00:00:00Z
+    Plan:
+      title: Plan
+      description: A simplified format for a plan
+      type: object
+      properties:
+        id:
+          description: The plan ID number
+          type: integer
+        name:
+          description: The name of the plan
           type: string
+      example:
+        id: 1
+        name: 'Basic Plan'
+    Account:
+      title: Account
+      description: Represents the base type for all accounts
+      type: object
+      properties:
+        id:
+          description: The account ID number
+          type: integer
         org_name:
           description: The organization name
           type: string
-        admin_name:
-          description: The name of the admin user
-          type: string
-        apps_count:
-          description: The total amount of applications
-          type: integer
       example:
-        id: 1
-        created_at: 2019-10-18T05:13:26Z
-        updated_at: 2019-10-18T05:13:27Z
-        state: approved
+        id: 0
         org_name: Umbrella Corp.
-        admin_name: Oswell E. Spencer
-        apps_count: 3
+    DeveloperAccount:
+      allOf:
+        - $ref: '#/components/schemas/Account'
+        - type: object
+          title: Root Type for developerAccountData
+          description: A simplified format for a developer account data.
+          properties:
+            created_at:
+              format: date-time
+              description: When it was created
+              type: string
+            updated_at:
+              format: date-time
+              description: When it was last updated
+              type: string
+            state:
+              description: The current state of the account
+              type: string
+            admin_name:
+              description: The name of the admin user
+              type: string
+            apps_count:
+              description: The total amount of applications
+              type: integer
+          example:
+            id: 1
+            created_at: 2019-10-18T05:13:26Z
+            updated_at: 2019-10-18T05:13:27Z
+            state: approved
+            org_name: Umbrella Corp.
+            admin_name: Oswell E. Spencer
+            apps_count: 3
   responses:
+    getApplications:
```

Comment:
It seems like this response is never used
And why not use `ApplicationsList` to keep it consistent, instead of `getApplications` that suggest an action

Reply:
You're absolutely right